plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/xsd/com.google.cloud.tools.eclipse.appengine.libraries.xsd

```diff
@@ -143,14 +139,18 @@ project, under WEB-INF/lib.
          <xs:attribute name="javadocUri" type="xs:anyURI">
             <xs:annotation>
                <xs:documentation>
-                  URI that can be used by the IDE to lookup javadoc for the classes contained in this artifact.
+                  URI that can be used by the IDE to lookup javadoc for the classes contained in this artifact. If
+undefined, an attempt will be made to download the javadoc artifact from the same repository where the binary artifact
+is downloaded from.
```

Comment:
That last bit is a bit wordy.
_If unspecified, the javadoc will be downloaded from the artifact location if available._
(suggestion only)